modules/platforms/dotnet/Apache.Ignite/Internal/Proto/BinaryTuple/BinaryTupleBuilder.cs

```diff
@@ -280,6 +344,22 @@ public void AppendFloat(float value)
             OnWrite();
         }
 
+        /// <summary>
+        /// Appends a gloat.
+        /// </summary>
+        /// <param name="value">Value.</param>
+        public void AppendFloatNullable(float? value)
```

Comment:
Just curious, why woudn't we go with method overloading rather than having a separate naming? I.e. AppendFloat(float? value)? 

To make it in sync with Get* methods? Like GetFloatNullable?

Reply:
I've considered this too, but decided to be consistent with `BinaryTupleBuilder` in Java, so it is easier to look at client implementations in different languages.